src/nomad_simulations/schema_packages/physical_property.py

```diff
@@ -146,109 +81,113 @@ class PhysicalProperty(ArchiveSection):
         """,
     )
 
+    physical_property_ref = Quantity(
+        type=Reference(SectionProxy('PhysicalProperty')),
+        description="""
+        Reference to the `PhysicalProperty` section from which the physical property was derived. If `physical_property_ref`
+        is populated, the quantity `is_derived` is set to True via normalization.
+        """,
+    )
+
     is_scf_converged = Quantity(
         type=bool,
         description="""
         Flag indicating whether the physical property is converged or not after a SCF process. This quantity is connected
         with `SelfConsistency` defined in the `numerical_settings.py` module.
         """,
-    )
+    )  # ? tie to calculation, not individual property
 
     self_consistency_ref = Quantity(
         type=SelfConsistency,
         description="""
         Reference to the `SelfConsistency` section that defines the numerical settings to converge the
         physical property (see numerical_settings.py).
         """,
+    )  # ? remove
```

Comment:
Not sure about removing, but at least move up to Outputs level? Similar to is_scf_converged

Reply:
The discussion regarding (SCF) convergence has been deferred to its own issue (https://github.com/nomad-coe/nomad-simulations/issues/207).